CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 - Serialisation added so models can be written to/read from JSON ([#3397](https://github.com/pybamm-team/PyBaMM/pull/3397))
 
 ## Bug fixes
-
+- Fixed Style checker on PRs does not actually check anything ([#3571](https://github.com/pybamm-team/PyBaMM/pull/3580))
```

Comment:
We can skip this, not a user-facing change.
```suggestion
```

Reply:
Reverted.